renderer/types.ts

```diff
@@ -6,10 +6,8 @@ declare global {
     interface PageContext {
       config: {
         pageStarted?: EventCallable<{ params: Record<string, string>; data: unknown }>;
+        pageClientStarted?: EventCallable<void>;
```

Comment:
What do you think to make universal interface with `pageStarted`, `pageClientStarted`, and possible `createServerStarted`?

```suggestion
        pageClientStarted?: EventCallable<{ params: Record<string, string>; data: unknown }>;
```